main.css

```diff
@@ -205,3 +205,87 @@ img {
   color: #344563;
   margin-top: 60px;
 }
+
+.footer {
+  background-color: #6070ff;
+  min-height: 779px;
+  max-width: 375px;
+  border-radius: 80px 0 0 0;
+  background-image: url(./images/background.png);
```

Comment:
- [ ] Kindly make sure to follow the design template as given in Figma. As you have noticed your background image is not properly placed as given. Fix it.
- Your design:
<img width="376" alt="Screenshot 2022-03-22 at 10 55 20 PM" src="https://user-images.githubusercontent.com/70269507/159566870-cd164ed2-69a8-46ed-9105-f33e7633ffa8.png">

- Figma design:
<img width="222" alt="Screenshot 2022-03-22 at 11 08 59 PM" src="https://user-images.githubusercontent.com/70269507/159567045-091ac131-1bb4-4b21-8f8e-0adf833ebb33.png">

Reply:
Done